demisto_sdk/commands/common/content/objects/pack_objects/pack_metadata/pack_metadata.py

```diff
@@ -27,7 +28,10 @@
 XSOAR_CERTIFIED = "certified"
 XSOAR_EULA_URL = "https://github.com/demisto/content/blob/master/LICENSE"
 
-CORE_PACKS_LIST = get_core_pack_list()
+if not is_sdk_defined_working_offline():
+    CORE_PACKS_LIST = get_core_pack_list()
+else:  # In case the sdk defined working offline (customers)
+    CORE_PACKS_LIST = []
```

Comment:
instead, use a `[]` default to `get_marketplace_to_core_packs`, to be used when offline